include/oneapi/tbb/detail/_flow_graph_impl.h

```diff
@@ -146,6 +146,54 @@ class graph_task : public d1::task {
     friend graph_task* prioritize_task(graph& g, graph_task& gt);
 };
 
+#if __TBB_PREVIEW_FLOW_GRAPH_TRY_PUT_AND_WAIT
+class graph_task_with_message_waiters : public graph_task {
```

Comment:
Perhaps, rename to something like:
```suggestion
class uniquely_awaited_graph_task : public graph_task {
```
Besides `uniquely` the following words seem to be more-or-less suitable: `separately`, `individually`, `independently`, `distinctly`.

Reply:
Personally, I like preserving something about waiting in the name. I think it is more obvious than tracking the message. And actually, it is not tracked separately from the entire graph, it just adds one more level of tracking for individual wait_contexts.

Reply:
> What was wrong with the original name?

It exposes the implementation details.

> And actually, it is not tracked separately from the entire graph, it just adds one more level of tracking for individual wait_contexts.

Perhaps, then `trackable_graph_task`?

Reply:
As discussed offline, renamed to `trackable_messages_graph_task`.